### PR TITLE
Added RBAC groups and namespace for slack infra

### PR DESCRIPTION
**Pull request by bartsmykla**
/cc @dims @ameukam 

Signed-off-by: Bart Smykla <bsmykla@vmware.com>

### CODE REVIEW

**Comment by stp-ip** [comment]
looks good from a code perspective and what we discussed in the call aka keeping separate namespaces.

File: groups/groups.yaml [hunk 1]
```diff
@@ -405,6 +409,57 @@ groups:
...
+
+  # Collective group for RBAC groups for slack infrastructure
+  # created to not duplicate members
+  - email-id: k8s-infra-slack-infra-admins@kubernetes.io
+    name: k8s-infra-slack-infra-admins
+    description: |-
+      ACL for Slack Infra Admins
+    settings:
+      ReconcileMembers: "true"
+    members:
```
Comment (ameukam) [inline groups/groups.yaml:456]: @spiffxp is not missing in this list ?

Comment (bartsmykla) [reply groups/groups.yaml:456]: Right, sorry :-D

Comment (bartsmykla) [reply groups/groups.yaml:456]: Fixed

**Comment by spiffxp** [comment]
ref: https://github.com/kubernetes/k8s.io/issues/751

**Comment by spiffxp** [comment]
FWIW I would have been fine if all this ran in a "slack-infra" namespace instead of giving each deployment/service its own namespace. It all currently runs in the default namespace of its cluster.

**Pull request merged**